conversions/conversions.py

```diff
@@ -15,53 +14,86 @@ class Conversions:
     MM_TO_IN_CONVERSION_FACTOR = 0.0393701
     ACCEL_G_CONSTANT = 1.0
 
-    def __init__(self, filename: str):
+    def __init__(self, filename: str, filename2: str):
         self.filename = filename
-        self.data = pd.DataFrame(pd.read_csv(filename))
-        self.switch_columns()
+        self.filename2 = filename2
+        self.data_linpot = pd.DataFrame(pd.read_csv(filename))
+        self.data_accel = pd.DataFrame(pd.read_csv(filename2))
 
-        print(self.data)
+        self.data_linpot = self.switch_columns(self.data_linpot)
 
-    def switch_columns(self):
-        self.data = self.data.rename(
+    def switch_columns(self, df: pd.DataFrame) -> pd.DataFrame:
+        df = df.rename(
             columns={
                 "Front Right": "Front Left",
                 "Front Left": "Rear Left",
                 "Rear Left": "Front Right",
             }
         )
+        return df
 
-    # converts voltage to mm and then inches for as spring rates are in inches / pound
-    def convert_voltage_to_in(self):
-        for i, row in self.data.iterrows():
-            self.data.loc[i, "Front Right"] = (-(row["Front Right"] * constants.LINPOT_CONVERSION_CONSTANT) +
-                                               constants.LINPOT_CONVERSION_OFFSET) * constants.MM_TO_IN_CONVERSION_FACTOR
-            self.data.loc[i, "Front Left"] = (-(row["Front Left"] * constants.LINPOT_CONVERSION_CONSTANT) +
-                                              constants.LINPOT_CONVERSION_OFFSET) * constants.MM_TO_IN_CONVERSION_FACTOR
-            self.data.loc[i, "Rear Right"] = (-(row["Rear Right"] * constants.LINPOT_CONVERSION_CONSTANT) +
-                                              constants.LINPOT_CONVERSION_OFFSET) * constants.MM_TO_IN_CONVERSION_FACTOR
-            self.data.loc[i, "Rear Left"] = (-(row["Rear Left"] * constants.LINPOT_CONVERSION_CONSTANT) +
-                                             constants.LINPOT_CONVERSION_OFFSET) * constants.MM_TO_IN_CONVERSION_FACTOR
+    # todo return modifed dataframe that is passed in as an argument
+    # todo change to new linpot orientation
+    def switch_columns2(self):
+        self.data_linpot = self.data_linpot.rename(
+            columns={
+                "Front Right": "Front Left",
+                "Front Left": "Rear Left",
+                "Rear Left": "Front Right",
+            }
+        )
 
-    # fill out method for converting voltage to mm
+    # converts voltage to mm
+    # todo return modifed dataframe that is passed in as an argument
     def convert_voltage_to_mm(self):
-        pass
+        for i, row in self.data_linpot.iterrows():
+            self.data_linpot.loc[i, "Front Right"] = (-(row["Front Right"] * constants.LINPOT_CONVERSION_CONSTANT) +
```

Comment:
you still seem to be modifying the lin pot file that is being passed in, instead make a copy of the lin pot file being passed in and return it

Reply:
This is the original one. The one that converts to inches does passes in a dataframe and returns the processed one.